src/lib/utils/logger.utils.ts

```diff
@@ -39,32 +39,6 @@ export class NGXLoggerUtils {
     return configColorScheme[level];
   }
 
```

Comment:
Isn't this function still necessary?

Reply:
Not sure what you are pointing out, it seems like you referenced `getColorFromConfig` func but I did not touch it and yes it is used by `getColor`

Reply:
Github shows that the function `getCallerDetails` is now removed in the commit

Reply:
yes there was 2 of those, one static that was not used anymore and I deleted it. The one used is still there in mapper.service.ts

Reply:
Ok cool